model_signing/model.py

```diff
@@ -145,17 +142,13 @@ def verify(self, inputfn: Path, signaturefn: Path,
             bundle_bytes = signaturefn.read_bytes()
             bundle = Bundle().from_json(bundle_bytes)
 
-            material: tuple[Path, VerificationMaterials]
             contentio = io.BytesIO(Serializer.serialize_v1(
```

Comment:
The `contentio` variable becomes unused now. Which means that we still don't serialize the model when we verify.